### PR TITLE
docs: container build error logs are now accessible to end users

### PR DESCRIPTION
**Pull request by AurelienGasser**
Companion PRs:
- https://github.com/Substra/substra-backend/pull/483 (main PR)
- https://github.com/Substra/orchestrator/pull/38
- https://github.com/Substra/substra/pull/295
- https://github.com/Substra/substra-tests/pull/218
- https://github.com/Substra/substra-frontend/pull/116
- https://github.com/Substra/substra-documentation/pull/184 (this PR)

---

### CODE REVIEW

**Comment by AurelienGasser** [comment]
Not sure who has write access so I requested a review from a bunch of people. Feel free to merge the PR.

**Review by AlexandrePicosson — APPROVED** [review]

**Pull request merged**
